### PR TITLE
IJoypadController.h: fix #1191

### LINKED ISSUE #1191: Inconsistency in documentation of IJoypadController::getButton method
In the documentation of the method in https://github.com/robotology/yarp/blob/devel/src/libYARP_dev/include/yarp/dev/IJoypadController.h#L97 , it seems that the button output is boolean (true/false) but apparently the output parameter of the method is actually a `float` . 

@aerydna

### PR DESCRIPTION
**Pull request by aerydna**
see #1191

### CODE REVIEW

File: src/libYARP_dev/include/yarp/dev/IJoypadController.h [hunk 1]
```diff
@@ -139,7 +143,7 @@ class  YARP_dev_API yarp::dev::IJoypadController
     /**
      Get the value of a touch if present, return false otherwise
     * @brief getTouch
-    * @param value a vector that will contain the joystick position. the size of the vector will give you the Number of degrees of freedom
+    * @param value a vector that will contain the touch values normalized from 0.0 to 1.0.
```
Comment (traversaro) [inline src/libYARP_dev/include/yarp/dev/IJoypadController.h]: I tested this on a real joypad, and the "touch" values seemd to be xy coordinates, is this correct?

Comment (aerydna) [inline src/libYARP_dev/include/yarp/dev/IJoypadController.h]: this is, indeed. what was your doubt? 
a touch Surface usually gives a xy position.. you can have also one dimensional touch Surface as well and/or boolean touch Surface (like the oculus touch buttons)

Comment (aerydna) [inline src/libYARP_dev/include/yarp/dev/IJoypadController.h]: maybe can be useful a method to get the touch Surface dimension.. or the resolution. i'l put it on the todo list

Comment (traversaro) [inline src/libYARP_dev/include/yarp/dev/IJoypadController.h]: Ah I got it, I just interpreted "touch values" as normal force measures. I think several users that are used to work with robotics skin could be confused by this, perhaps an additional sentence to explain what "touch values" would be beneficial.

Comment (randaz81) [inline src/libYARP_dev/include/yarp/dev/IJoypadController.h]: `yarp::math::Vec2D` could be a better type for value

**Pull request merged**